packages/xrpl/src/models/methods/ammInfo.ts

```diff
@@ -0,0 +1,142 @@
+import { Amount, Issue, IssuedCurrencyAmount } from '../common'
+
+import { BaseRequest, BaseResponse } from './baseMethod'
+
+/**
+ * The `amm_info` command retrieves information about an AMM instance.
+ * Returns an {@link AMMInfoResponse}.
+ *
+ * @category Requests
+ */
+export interface AMMInfoRequest extends BaseRequest {
+  command: 'amm_info'
+
+  /**
+   * Specifies one of the pool assets (XRP or token) of the AMM instance.
+   * Both asset and asset2 must be defined to specify an AMM instance.
+   */
+  asset?: Issue
+
+  /**
+   * Specifies the other pool asset of the AMM instance.
+   * Both asset and asset2 must be defined to specify an AMM instance.
+   */
+  asset2?: Issue
+}
+
+interface AuthAccount {
+  account: string
+}
+
+interface VoteEntry {
+  account: string
+  trading_fee: number
+  vote_weight: number
+}
+
+/**
+ * Response expected from an {@link AMMInfoRequest}.
+ *
+ * @category Responses
+ */
+export interface AMMInfoResponse extends BaseResponse {
+  result: {
+    amm: {
+      /**
+       * The account that tracks the balance of LPTokens between the AMM instance via Trustline.
+       */
+      amm_account: string
+
+      /**
+       * One of the pool assets (XRP or token) of the AMM instance.
+       */
+      amount: Amount
+
+      /**
+       * The other pool asset of the AMM instance.
+       */
+      amount2: Amount
+
+      /**
+       * Flag indicating whether asset2 is frozen.
+       */
+      asset2_frozen: boolean
```

Comment:
`asset_frozen` field does not exist

https://xrpl.org/ja/amm_info.html#amm-description-object

Reply:
Ah good catch, I'll add this. Thanks 🙂